source/models/customizing-adapters.md

```diff
@@ -69,6 +69,54 @@ most commonly extended adapter that ships with Ember Data. It has a
 handful of hooks that are commonly used to extend it to work with
 non-standard backends.
 
```

Comment:
What do you think about extracting this into a separate file, called customizing the RestAdapter? And here just have intro to the adapter api and links to popular adapters.